tests/perf/test_detection.py

```diff
@@ -20,7 +20,7 @@
 def fxt_deterministic(request: pytest.FixtureRequest) -> bool:
     """Override the deterministic setting for detection task."""
     deterministic = request.config.getoption("--deterministic")
-    deterministic = True if deterministic is None else deterministic == "true"
+    deterministic = "warn" if deterministic is None or deterministic == "warn" else deterministic == "true"
```

Comment:
Could you update argparse part in tests/conftest.py?

Reply:
Done https://github.com/openvinotoolkit/training_extensions/pull/3741/commits/2d5cef1ef2bbc576bebd10fd1a96e812d6ef0548

Reply:
Ah, I mean the function below at "tests/conftest.py"
```
    parser.addoption(
        "--deterministic",
        choices=["true", "false"],
        default=None,
        help="Turn on deterministic training (true/false).",
    )
```

Reply:
Done https://github.com/openvinotoolkit/training_extensions/pull/3741/commits/28837861e6f6151bef8608a940b020bc6ed22437